### ISSUE TITLE
Setting up Google Workspace for Nonprofits

### ISSUE THREAD

**Issue by anisa-hawes** [programminghistorian/jekyll#2492]
Following discussions arising from #1981, we have taken the first steps towards setting up a Google Workspace for Nonprofits.

It seems sensible to open a separate Issue, as this represents a distinct task.

An initial part of the set up involves verification with Tech Soup / Charity Digital. We've hit an obstacle here, because the Charity Commission's Database (linked from Charity Digital) is currently unavailable. 

We've tried contacting them by phone, and have now sent an email to seek help. We'll post a further comment here with an update when we have one.

**Comment by drjwbaker** [comment]
Is https://register-of-charities.charitycommission.gov.uk/charity-search/-/charity-details/5181272/charity-overview not sufficient?

**Comment by acrymble** [comment]
@drjwbaker it's an issue with the Charity Digital website not working.

**Comment by acrymble** [comment]
@drjwbaker I got in but they need us to provide:

> If your organisations annual income is not shown on your entry on your registrar’s (Charity Commission, OSCR etc.) website, please attach a copy of your most recent annual accounts.

> If you are a recently set up charity and you are yet to publish your accounts, please provide a statement of your project income for your first year of operation. This should be on your charity's letterheaded paper and certified by an officer or trustee.

Can you tell me which document in the business archive is most appropriate? This is a verification thing.

**Comment by drjwbaker** [comment]
Thus far we've only filled accounts for 2019-2020, which are here https://find-and-update.company-information.service.gov.uk/company/12192946/filing-history (entry for 02 Mar 2021)

**Comment by acrymble** [comment]
@anisa-hawes we should get email confirmation when they check our documents. Then we can get the TechSoup validation token we need to proceed with registration at Google.

**Comment by acrymble** [comment]
Have followed up. They wanted more info. An email got lost. Hopefully soon we can get in and get the TechSoup code.

**Comment by acrymble** [comment]
The account is now verified and we have a Google for Nonprofits account. I haven't added any services yet @anisa-hawes

**Comment by acrymble** [comment]
Because I already tried to do this via the paid route back in the summer, the domain is "already in use" so I've had to contact Google to help unlink it so we can re-apply it to this nonprofit account. That should then allow me to request the DNS verification code I need to prove it's our domain, [steps here...](https://support.google.com/a/answer/6248925?hl=en&ref_topic=4487622#zippy=%2Cim-stuck-and-need-help%2Cim-not-sure-what-to-do-next).

Once that is done we can add services (eg emails).

**Comment by drjwbaker** [comment]
Is this clashing with https://github.com/programminghistorian/jekyll/issues/1981#issuecomment-1049447843 ?

**Comment by acrymble** [comment]
I don't think so. i think it's clashing with the fact that I can't find the passwords for the admin@programminghistorian.org account I had tried to set up. So I'm getting them to wipe it so we can start over.

**Comment by acrymble** [comment]
I've managed to verify our domain now. This apparently has to be checked by a human and we should hear back within 3 days if everything is ok, but I think we're getting there.

**Comment by acrymble** [comment]
@anisa-hawes I think this is at the stage where we can get you logged in to see if you can set up the services you want to use.

**Comment by acrymble** [comment]
@anisa-hawes has been given the login details and is exploring this service.
